crates/biome_cli/src/execute/migrate/prettier.rs

```diff
@@ -278,7 +280,12 @@ impl TryFrom<PrettierConfiguration> for biome_configuration::Configuration {
         if !value.overrides.is_empty() {
             let mut overrides = biome_configuration::Overrides::default();
             for override_elt in value.overrides {
-                overrides.0.push(override_elt.try_into()?);
+                let sources = override_elt.files.clone();
+                overrides.0.push(
+                    override_elt
+                        .try_into()
+                        .with_context(|| format!("override element matching {sources:?}"))?,
```

Comment:
Even this change doesn't seem related to the fix because this new message doesn't appear in any of the tests we have. Or, if there aren't, we should add them.